test/index.test.js

```diff
@@ -28,6 +28,37 @@ test('[useCallback] sinon-style', function(assert) {
   });
 });
 
+test('[useCallback] sinon-style onCall', function(assert) {
+  var getObject = AWS.stub('S3', 'getObject');
+  getObject.onCall(0).yields(null, data);
```

Comment:
On second glance, it appears `onCall` works just fine. I didn't see it in the list of methods but it appears to be working as expected, with the `stub.onCall(0).yields()` pattern. This PR isn't really necessary, then.